docs/metrics.md

```diff
@@ -144,14 +96,16 @@ See the Events section for specific examples of event registration and recording
 
 ## Metrics Overview
 
+(this section may redundant with analysis section)
 For alpha, we'd like to (ideally) like to be able to track the following general categories of things:
 
 -   The setup flow, so we can know at what points (if any) people quit the flow before finishing it
--   Top-level interactions centered around use of the Lockbox toolbar icon. This includes interactions within the initial pop-over that is displayed when the user clicks the icon.
--   Interactions with the main CRUD editor view which lists the user's Lockbox items (credentials)
+-   Top-level interactions centered around use of the Lockbox toolbar icon. This includes interactions within the initial doorhanger that is displayed when the user clicks the icon.
+-   Interactions with the list of the user's Lockbox items (credentials)
 -   Interactions with the add / modify dialogs used to enter / edit item information
 -   Changes to the datastore that actually contains the user's items, in addition to user actions that lead to those changes
 -   When the user submits feedback about lockbox
+-		Usage of the copy and reveal functions for stored Lockbox items.
```

Comment:
Indenting is wacky here, causing this to be converted into a code block. 🤷‍♀️

Reply:
Thanks. I'll go fix this right now @pdehaan @irrationalagent